tests/NRedisStack.Tests/Search/MissingEmptyValuesSearchTests.cs

```diff
@@ -0,0 +1,123 @@
+using StackExchange.Redis;
+using NRedisStack.Search;
+using NRedisStack.RedisStackCommands;
+using Xunit;
+using NRedisStack.Search.Literals;
+using NetTopologySuite.Geometries;
+
+namespace NRedisStack.Tests.Search;
+
+public class MissingEmptyValuesSearchTests : AbstractNRedisStackTest, IDisposable
```

Comment:
These tests should be added in existing test file.
Or, the file name should be more generic like IndexCreationTests, IndexCreationArgumentsTests, etc.

Reply:
my intention is to disassemble the existing one. So adding it more would not help for next steps. 
I agree about the naming and will pick a better one.